### DISCUSSION TITLE
why does Stylus need to read my browsing history?

### DISCUSSION THREAD

**Discussion by ghost** [openstyles/stylus#1776] [Q&A] [answered]
so, I was looking at the extension page thingy for stylus and I've just realized, It apparently needs to read my browser history?

![image](https://github.com/openstyles/stylus/assets/72381315/65cc96b8-978e-43a1-bd6b-dada26d58ab4)

**Comment by tophf** [comment, marked as answer]
It doesn't. The phrasing of this permission is just incorrect and it was reported to chromium team multiple times. Stylus observes the change in the URL of a tab to apply the matching styles to the tab.

**Reply by ghost** [reply]
I see, I agree with you on the phrasing part, since having that permission doesn't mean a extension is storing your browsing history.

(anyway, thank you so much)
